docs/docs/platform/layouts.md

```diff
@@ -76,55 +111,41 @@ const updatedLayout = await novu.layouts.update(layoutId, {
 });
 ```
 
-And to set a layout as default programmatically it is as easy as to execute:
+### Set default layout
+
+When executing this, the existing default layout in the environment will be automatically set as non default and the chosen layout will be set as default. This action is non reversible.
 
 ```typescript
 const layoutId: LayoutId = '<LAYOUT_ID>'; // The unique identifier of the layout to be set as default.
 
 await novu.layouts.setDefault(layoutId);
 ```
 
-When executing this, the existing default layout in the environment will be automatically set as non default and the chosen layout will be set as default. This action is non reversible.
+### Delete an existing layout
 
-## Assign Layout to template
-
-To assign, choose a layout through the email editor.
-You can preview your layout combined with your email content through the `Preview` tab.
-
-   <div align="center">
-     <picture>
-       <source media="(prefers-color-scheme: dark)" srcset="/img/platform/layouts/dark-assign-layout.png"/>
-       <img src="/img/platform/layouts/light-assign-layout.png" width="680" alt="Logo"/>
-     </picture>
-   </div>
```

Comment:
Any reason the screenshot is deleted from the docs?

Reply:
He just changed its location to a new section